src/tcgwars/logic/impl/gen8/AmazingVoltTackle.groovy

```diff
@@ -237,16 +235,19 @@ public enum AmazingVoltTackle implements LogicCardInfo {
       return evolution (this, from:"Nuzleaf", hp:HP150, type:G, retreatCost:3) {
         weakness R
         bwAbility "Tengu Double", {
+          // TODO: Implement abiltiy
           text "The effect of each Supporter card in your opponent's hand becomes 'Draw 3 cards'."
           actionA {
           }
         }
         move "Fan Tornado", {
           text "110 damage. You may have your opponent switch their Active Pokémon with 1 of their Benched Pokémon."
           energyCost G, C
-          attackRequirement {}
           onAttack {
             damage 110
+            bg.deterministicCurrentThreadPlayerType=bg.currentTurn.opposite
+            switchYourActive()
+            bg.clearDeterministicCurrentThreadPlayerType()
```

Comment:
I'd refrain from using the deterministicCurrentThreadPlayerType setting, it should be meant for internal engine use only.

Reply:
Right. Whirlwind static is the proper thing to use here. I just couldn't remember that it existed when I was writing this. Will fix.